### PR TITLE
Update Gradle to 7.5

### PR DESCRIPTION
**Pull request by KCeh**
I had troubles with publishing to mavenLocal and using Gradle in general so I updated it

### CODE REVIEW

**Comment by KCeh** [comment]
I will look into why Bitrise is falling

File: prince-of-versions/quality.gradle [hunk 1]
```diff
@@ -17,29 +16,6 @@ task checkstyle(type: Checkstyle) {
     classpath = files()
 }
 
-task findbugs(type: FindBugs) {
```
Comment (antunflas) [inline prince-of-versions/quality.gradle:20]: Why did you remove findbugs? If it cannot be used anymore we can replace it with Spotbugs, which should be effectively the same.

**Review by antunflas — APPROVED** [review]

File: prince-of-versions/quality.gradle (hunk 1 above)
Comment (KCeh) [reply prince-of-versions/quality.gradle:20]: 👌, will do

File: build.gradle [hunk 2]
```diff
@@ -3,17 +3,25 @@ apply from: 'versions.gradle'
...
+        classpath 'com.android.tools.build:gradle:7.4.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "com.github.spotbugs.snom:spotbugs-gradle-plugin:5.2.5"
     }
 }
 
+ext {
+    SpotBugsTask = com.github.spotbugs.snom.SpotBugsTask
+}
+
```
Comment (KCeh) [inline build.gradle:24]: Gradle gave me headaches when configuring Spotbugs in quality scripts, this is only workaround I found
